unison-src/transcripts-manual/benchmarks/array-sort.u

```diff
@@ -10,6 +10,7 @@ cmp m n = match compare m n with
   -1 -> Less
   +0 -> Equal
   +1 -> Greater
+  _ -> Less
```

Comment:
with pattern coverage checking, this case is now needed (though it will never be hit in practice)